overview.md

```
@@ -1,141 +1,234 @@
-https://mermaid.live
+# RERO MEF Data Model Overview
+
+View and edit diagrams at: https://mermaid.live
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)